gui/wxpython/startup/utils.py

```diff
@@ -21,38 +21,47 @@
 
 
 def get_possible_database_path():
-    """Finds a path to what is possibly a GRASS Database. 
-
-    Looks for directory named grassdata in the usual locations.
-
-    Returns the path as a string or None if nothing was found, so the
-    return value can be used to test if the directory was found.
+    """Finds the directory for possible GRASS Database.
```

Comment:
The description is awkward. I suggest:

    """Looks for directory 'grassdata' (case-insensitive)
    in standard locations to detect existing GRASS Database.
    Returns the path as a string or None if nothing was found."""